MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -119,7 +119,11 @@ class RouteManeuverViewController: UIViewController {
     
     public func updateStreetNameForStep() {
         if let name = step?.names?.first {
-            streetLabel.unabridgedText = name
+            if let step = step, let ref = step.codes?.first, ["i", "us"].contains(ref.lowercased().components(separatedBy: " ").first!) {
```

Comment:
Unfortunately, the Directions API doesn’t provide any information about the “type” of a road segment. #297 for example must query the map’s vector tiles for this information, but it can only do so for features currently visible on the map, not roads that may be far out of view.